content/search/results.js

```diff
@@ -16,21 +16,28 @@
     var $ul = document.createElement('ul');
 
     $h1.innerText = 'Search results';
+    $ul.className = 'articles-list';
```

Comment:
I noticed that there was already some styling to add nicer bullets to search results.